services/IoT/applications/libraries/nodejs.md

```diff
@@ -97,18 +97,18 @@ To connect to {{site.data.keyword.iot_short_notm}}, submit a *connect* request,
 	});
 ```
 
-After successfully connecting to the {{site.data.keyword.iot_short_notm}} service, the application client sends a *connect* event. So all the logic can be implemented inside this callback function.
+After successfully connecting to the {{site.data.keyword.iot_short_notm}} service, the application client sends a *connect* event, so all the logic can be implemented inside this callback function.
 
 
 
-The application client automatically tries to reconnect when it loses connection. When the reconnection is successful, the client emits a *reconnect* event.
+The application client automatically tries to reconnect when it loses connection. When the reconnection is successful, the client sends a *reconnect* event.
```

Comment:
changed emits to sends to simplify and streamline language. calling it out in case "emit" is a legit thing in node.js

Have made the change elsewhere in this document as well.